src/main.ts

```diff
@@ -62,6 +63,7 @@ app.config.globalProperties.$filters = {
   }
 }
 
+init(store.getters['user/getUserToken'], store.getters['user/getInstanceUrl'], 3000)
```

Comment:
Why we are passing 3000?

Reply:
It's the maxAge for cache, will update it to use the env entry VUE_APP_CACHE_MAX_AGE